Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rust:1.29.2
+
+RUN apt-get update -y && mkdir /target
+
+WORKDIR /usr/src/crust
+COPY . .
+RUN cargo build --release --verbose --target-dir /target
```

Comment:
I think we can also pre-build the tests and it should save us a bit more CI time.
It should be sufficient to add just a single command:
```
RUN cargo build --tests --release --verbose --target-dir /target 
```

Reply:
Great, thanks Nikita! I'll do that just now.